datadog_sync/utils/filter.py

```diff
@@ -89,10 +97,25 @@ def process_filters(filter_list: List[str]) -> Dict[str, List[Filter]]:
         if invalid_filter:
             continue
 
-        # Build and assign regex matcher to VALUE key
-        f_dict[FILTER_VALUE] = build_regex(f_dict)
-
-        f_instance = Filter(f_dict[FILTER_TYPE].lower(), f_dict[FILTER_NAME], f_dict[FILTER_VALUE])
+        # We are setting ExactMatch as the default as this lines up with the future plans to use the Value Key
+        # directly as the regex for the filter
```

Comment:
```suggestion
        #  Default to EXACT_MATCH_OPERATOR for backward compatibility. This behavior will be removed in the
        # future as it can already be achieved using regex in the Value.
```